site/content/resources/compatibility-matrix.md

```diff
@@ -11,6 +11,7 @@ These combinations of versions are specifically tested in CI and supported by th
 | Contour Version | Envoy Version        | Kubernetes Versions | Operator Version | Gateway API Version |
 | --------------- | :------------------- | ------------------- | ---------------- | --------------------|
 | main            | [1.27.0][41]         | 1.28, 1.27, 1.26    | N/A              | v1alpha2, v1beta1   |
+| 1.26.0          | [1.27.0][41]         | 1.28, 1.27, 1.26    | N/A              | v1alpha2, v1beta1   |
```

Comment:
I was thinking maybe we should start listing GWAPI release versions instead of just the API versions, might be a little more meaningful, but can consider doing that update separately.

Reply:
yeah thats a good idea!